### DISCUSSION TITLE
Email Notification of assigned task.

### DISCUSSION THREAD

**Discussion by KernOil** [Budibase/budibase#2971] [Ideas]
I am trying to find a solution for when a task is assigned to an individual they will get a notification of the new assignment. 

how I think it should work is: in the automation when a row is updated it will look at a specific box to see if that box changed from x to y. If Y then read from the corresponding box to get the person's email and send them a notification.  but if the box does not change then no email will be sent. 

Does anyone have any ideas on how I can accomplish this?

**Comment by mjashanks** [comment]
This is currently not possible, although I'd also like it to be!

The reason is that we have no way of detecting a "change". The `Update Row` automation currently has access to the updated version of the row, not the previous version.

I think we should add this, but I'm not sure when we can.

**Reply by KernOil** [reply]
No worries, thank you for the update. I think this would be a good feature down the line.
